src/pages/index.js

```diff
@@ -0,0 +1,3 @@
+import TextFieldDemo from './TextFieldDemo/TextFieldDemo';
+
+export default TextFieldDemo;
```

Comment:
One line import export can be used here and in other files also.
export { default as TextField } from './TextField/TextField';